tests/integration_tests/test_postprocessing.py

```diff
@@ -131,7 +159,48 @@ def test_binary_predictions_with_number_dtype(tmpdir, backend, distinct_values):
         assert np.allclose(prob_0, 1 - prob_1)
 
 
-def predict_with_backend(tmpdir, config, data_csv_path, backend, num_predict_samples=None):
+@pytest.mark.parametrize("pct_positive", [1.0, 0.5, 0.0])
+def test_set_feature_saving(tmpdir, pct_positive):
+    backend = "local"
+    input_features = [
+        category_feature(vocab_size=3),
```

Comment:
Since `goemotions` is a text datasets, would it be possible to use `text_feature` in [tests.integration_tests.utils](https://github.com/ludwig-ai/ludwig/blob/3785ed5b53c83ea5e3e4c2df33ffd5677758896c/tests/integration_tests/utils.py#L234)?

Reply:
Done!